site/js/modules/webui/input.js

```diff
@@ -370,6 +371,10 @@ class FieldInstance {
             value = this.input.value;
         }
 
+        if (value?.length == 0) {
```

Comment:
Doesn't this break on a checkbox?
(If not syntactically, then surely semantically).